code/game/turfs/turf_changing.dm

```diff
@@ -100,7 +102,7 @@
 		else
 			lighting_clear_overlay()
 
-	update_openspace()
+	T.update_openspace()
```

Comment:
There were a lot of bugs with newly created openspace tiles not updating and just being black
they are gone now